march_hardware/src/joint.cpp

```diff
@@ -131,6 +131,15 @@ double Joint::getVelocity() const
   return this->velocity_;
 }
 
+float Joint::getVoltageVelocity() const
+{
+  float R_motor = 0.05;
+  float Kv_motor = 355;
+  float rpmToRad = M_PI / 30;
+  float Ke = Kv_motor * rpmToRad;
+  return (this->imc_->getMotorVoltage() + this->imc_->getMotorCurrent() * R_motor) / Ke;
+}
```

Comment:
Usually it is best to use `double` instead of `float`. This gives higher precision for (almost) no additional cost. Also because all inputs for controllers are `double`. It doesn't matter if `getMotorVoltage` and `getMotorCurrent` return `float`, since those will be automatically converted to `double`.